src/services/requestService.js

```diff
@@ -116,6 +116,71 @@ class RequestService {
       throw error;
     }
   }
+
+  /**
+   * Get requests by id
+   * @param {string} id to be created
+   * @return {object} Oject of request if found
+   */
+  static async findRequestsById(id) {
+    try {
+      const requests = await Requests.findOne({
+        where: { id }
+      });
+
+      return requests;
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  /**
+   * gets the Request by ID.
+   * @param {string} request The request object.
+   * @param {number} id The ID of the request.
+   * @returns {object} The  object.
+   */
+  static async updateRequest(request, id) {
+    try {
```

Comment:
can we change the param from request from data. Mainly because that's the data we will be updating our request with